packages/dev/core/src/Meshes/GreasedLine/greasedLineBaseMesh.ts

```diff
@@ -0,0 +1,358 @@
+import type { Scene } from "../../scene";
+import type { IGreasedLineMaterial } from "../../Materials/GreasedLine/greasedLineMaterialInterfaces";
+import { GreasedLinePluginMaterial } from "../../Materials/GreasedLine/greasedLinePluginMaterial";
+import { Mesh } from "../mesh";
+import { Buffer } from "../../Buffers/buffer";
+import type { Vector3 } from "../../Maths/math.vector";
+import { VertexData } from "../mesh.vertexData";
+import { DeepCopier } from "../../Misc/deepCopier";
+import { GreasedLineSimpleMaterial } from "../../Materials/GreasedLine/greasedLineSimpleMaterial";
+import type { Engine } from "../../Engines/engine";
+
+export enum GreasedLineRibbonPointsMode {

```

Comment:
Could you add a comment on this enum and the two following ones?

I ll merge after

Reply:
Sure. Added.